app/javascript/src/Common/SearchableSelect.scss

```diff
@@ -0,0 +1,50 @@
+// From colors.scss
+$highlight-color: #0088ce;
```

Comment:
I don't like we are repeating the same styles in both rails `app/assets/stylesheets` and webpacker `javascript/src` directories.
Maybe we can import them:
`@import 'app/assets/stylesheets/provider/_colors.scss';`
WDYT @didierofrivia @thomasmaas ?

Reply:
hmmm... have you tried it? Also, the correct way would be mainting a common one within the scope of webpack

Reply:
It would be wise not to repeat ourselves

Reply:
I tried locally, and it worked.
But we need to investigate deeper.
I think we can keep it in this PR, but open an Issue to do further research.
I will open it

Reply:
I agree there should be a common file for this, at the time of this PR there was only this scss file I think. I will create a _colors doppelganger for our React styles.